File: fizz.py
#!/usr/bin/env python3 

def main():
    
    for i in range(1, 101):

        if i % 3 == 0 and i % 5 == 0:
            print("Fizz Buzz")
            continue

        elif i % 3 == 0:
            print("Fizz")
            continue

        elif i % 5 == 0:
            print("Buzz")
            continue

        else: 
            print(i)


if __name__ == "__main__":
    main()

File: fizz.cc
#include <iostream>

void fizzbuzzUno();

int main(int argc, char **argv)
{
    fizzbuzzUno();
    return 0;
}

void fizzbuzzUno() 
{
    for(int i = 1; i < 100; ++i) {

        if((i % 3) == 0)
            std::cout << "Fizz" << "\n";

        else if((i % 5) == 0)
            std::cout << "Buzz" << "\n";

        else if((i % 15) == 0)
            std::cout << "Fizz Buzz" << "\n";

        else 
            std::cout << i << "\n";
    }
}

